src/command_modules/azure-cli-sql/HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+2.0.16
+++++++
+* Added sql server conn-policy show/update commands.
```

Comment:
This will need to be rebased to ensure that the data from all three PRs gets included correctly (git is notorious for screwing these up).

Reply:
Done